lib/wikitrivia_web/templates/layout/app.html.eex

```diff
@@ -30,6 +30,6 @@
       </main>
 
     </div> <!-- /container -->
-    <script src="<%= static_path(@conn, "/js/app.js") %>"></script>
+    <!-- <script src="<%= static_path(@conn, "/js/app.js") %>"></script> -->
```

Comment:
@alexcavalli The `app.js` file is the entrypoint to the SPA which I moved over to start when the user gets to the `/game/:game_id` route, aka the `show.html.eex` page. This was to prevent the SPA from being loaded on the `/game/new` route.